packages/cli/src/__tests__/workflow-runner.test.ts

```diff
@@ -178,7 +179,7 @@ describe('run', () => {
 		const permissionChecker = Container.get(CredentialsPermissionChecker);
 		jest.spyOn(permissionChecker, 'check').mockResolvedValueOnce();
 
-		jest.spyOn(WorkflowExecute.prototype, 'processRunExecutionData').mockReturnValueOnce(
+		jest.spyOn(WorkflowExecute.prototype, 'runPartialWorkflow2').mockReturnValueOnce(
```

Comment:
I'm not quite sure why it's calling `runPartialWorkflow2` now instead and if this is correct. Do you know?

Reply:
updated the test to hit the right branch, this test is about making sure that we don't hit the partial execution branch, but do a full execution instead

31422e4785ee2fdb6ff31fe0ecc289aa9f5b3181